com.darhon.drfinance.yml

```diff
@@ -0,0 +1,23 @@
+app-id: com.darhon.drfinance
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: drfinance
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --filesystem=host
```

Comment:
why `host`

Reply:
It is mainly to allow users to access an external drive (USB).